src/PatientChecking/PatientChecking/Controllers/AddressesController.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using PatientChecking.Services.ServiceModels;
+
+namespace PatientChecking.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class AddressesController : ControllerBase
+    {
+        private readonly PatientCheckInContext _context;
+
+        public AddressesController(PatientCheckInContext context)
+        {
+            _context = context;
+        }
+
+        // GET: api/Addresses
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<Address>>> GetAddresses()
+        {
+            return await _context.Addresses.ToListAsync();
+        }
+
+        // GET: api/Addresses/5
+        [HttpGet("{id}")]
+        public async Task<ActionResult<Address>> GetAddress(int id)
+        {
+            var address = await _context.Addresses.FindAsync(id);
+
+            if (address == null)
+            {
+                return NotFound();
+            }
+
+            return address;
+        }
+
+        // PUT: api/Addresses/5
+        // To protect from overposting attacks, see https://go.microsoft.com/fwlink/?linkid=2123754
+        [HttpPut("{id}")]
+        public async Task<IActionResult> PutAddress(int id, Address address)
```

Comment:
Should name: `UpdateAddress` . The meaningful word should be better than using the HTTP verb Put. Please review and update other places (PostAddress, PutPatient, PostPatient...)

Reply:
Because of initialization, all controllers are generated for testing, so they are not optimized.  I renamed with the meaningful words